girder/models/folder.py

```diff
@@ -544,6 +550,17 @@ def updateFolder(self, folder):
         # Validate and save the folder
         return self.save(folder)
 
+    def filter(self, doc, user=None, additionalKeys=None):
```

Comment:
Like in item.py, this function solves the issue of the meta field not showing up on a folder with no metadata when hitting the base /folder endpoint. However, the empty meta field is not saved here, so while consistent to the user, internally under this circumstance the meta tag can't be relied on. If you guys have any thoughts on if we should change this they're welcome.

Reply:
I agree that this isn't ideal, but I can't think of anything better myself either.

@girder/developers Let us know if you have better suggestions.